Rakefile

```diff
@@ -13,6 +14,18 @@ task :changelog, [:version] do |_t, args|
   sh "./scripts/generate_changelog.rb #{args[:version]}"
 end
 
+namespace :pl do
+  desc 'build the gem and place it at the directory root'
+  task :gem_build do
+    stdout, stderr, status = Open3.capture3('gem build facter.gemspec')
+    if !status.exitstatus.zero?
+      puts "Error building facter.gemspec \n#{stdout} \n#{stderr}"
```

Comment:
Should we just `exit(1)` or should we depend on CI to handle this errors in gem building?

Reply:
exit(1) or raise seems fine, I don't know if there's a preference for rake tasks?